client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -82,6 +82,11 @@ export default React.createClass( {
 		if ( ! this.isMounted() ) {
 			return;
 		}
+
+		if ( ! this.needsFax() ) {
+			delete form.fax;
```

Comment:
This will blank the `fax`. If we have it, and register a `.com` and then try to register a domain that needs a fax again, it will be blank. If that's ok, ready for merge.